sycl/test-e2e/AddressSanitizer/bad-context/bad-context-1.cpp

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: linux, cpu
```

Comment:
could you rename files in this folder as well and remove 1/2/3? Also see above bad-context-1.cpp and below double-free-1.cpp, local-overflow-1.cpp

Reply:
done